memlab.bib

```diff
@@ -9196,7 +9276,7 @@ @article{BernEtal11
 	Title = {A reservoir of time constants for memory traces in cortical neurons.},
 	Volume = {14},
 	Year = {2011},
-	Bdsk-File-1 = {YnBsaXN0MDDSAQIDBFxyZWxhdGl2ZVBhdGhZYWxpYXNEYXRhXxAiLi4vLi4vLi4vcGRmL0Jlcm5TZW9MZWVXYW5nMTFhLnBkZk8RAZIAAAAAAZIAAgAADE1hY2ludG9zaCBIRAAAAAAAAAAAAAAAAAAAAMq06O1IKwAAABM+6xVCZXJuU2VvTGVlV2FuZzExYS5wZGYAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAE0Eiy968NAAAAAAAAAAAAAMAAgAACSAAAAAAAAAAAAAAAAAAAAADcGRmAAAQAAgAAMq1IS0AAAARAAgAAMve9HQAAAABABAAEz7rAAwONwAL+iEAAJlgAAIAOU1hY2ludG9zaCBIRDpVc2VyczoAbWFyYzoAZG9jOgBwZGY6AEJlcm5TZW9MZWVXYW5nMTFhLnBkZgAADgAsABUAQgBlAHIAbgBTAGUAbwBMAGUAZQBXAGEAbgBnADEAMQBhAC4AcABkAGYADwAaAAwATQBhAGMAaQBuAHQAbwBzAGgAIABIAEQAEgAoVXNlcnMvbWFyYy9kb2MvcGRmL0Jlcm5TZW9MZWVXYW5nMTFhLnBkZgATAAEvAAAVAAIAC///AAAACAANABoAJABJAAAAAAAAAgEAAAAAAAAABQAAAAAAAAAAAAAAAAAAAd8=}}
+	Bdsk-File-1 = {YnBsaXN0MDDUAQIDBAUGJCVYJHZlcnNpb25YJG9iamVjdHNZJGFyY2hpdmVyVCR0b3ASAAGGoKgHCBMUFRYaIVUkbnVsbNMJCgsMDxJXTlMua2V5c1pOUy5vYmplY3RzViRjbGFzc6INDoACgAOiEBGABIAFgAdccmVsYXRpdmVQYXRoWWFsaWFzRGF0YV8QIi4uLy4uLy4uL3BkZi9CZXJuU2VvTGVlV2FuZzExYS5wZGbSFwsYGVdOUy5kYXRhTxEBkgAAAAABkgACAAAMTWFjaW50b3NoIEhEAAAAAAAAAAAAAAAAAAAAyrTo7UgrAAAAEz7rFUJlcm5TZW9MZWVXYW5nMTFhLnBkZgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAATQSLL3rw0AAAAAAAAAAAAAwACAAAJIAAAAAAAAAAAAAAAAAAAAANwZGYAABAACAAAyrUhLQAAABEACAAAy970dAAAAAEAEAATPusADA43AAv6IQAAmWAAAgA5TWFjaW50b3NoIEhEOlVzZXJzOgBtYXJjOgBkb2M6AHBkZjoAQmVyblNlb0xlZVdhbmcxMWEucGRmAAAOACwAFQBCAGUAcgBuAFMAZQBvAEwAZQBlAFcAYQBuAGcAMQAxAGEALgBwAGQAZgAPABoADABNAGEAYwBpAG4AdABvAHMAaAAgAEgARAASAChVc2Vycy9tYXJjL2RvYy9wZGYvQmVyblNlb0xlZVdhbmcxMWEucGRmABMAAS8AABUAAgAL//8AAIAG0hscHR5aJGNsYXNzbmFtZVgkY2xhc3Nlc11OU011dGFibGVEYXRhox0fIFZOU0RhdGFYTlNPYmplY3TSGxwiI1xOU0RpY3Rpb25hcnmiIiBfEA9OU0tleWVkQXJjaGl2ZXLRJidUcm9vdIABAAgAEQAaACMALQAyADcAQABGAE0AVQBgAGcAagBsAG4AcQBzAHUAdwCEAI4AswC4AMACVgJYAl0CaAJxAn8CgwKKApMCmAKlAqgCugK9AsIAAAAAAAACAQAAAAAAAAAoAAAAAAAAAAAAAAAAAAACxA==}}
```

Comment:
Not sure what this is...should remove

Reply:
file associated with BernEtal11. Removed.